cytodynamics-test-container/src/test/java/com/linkedin/cytodynamics/test/TestDynamicLoad.java

```diff
@@ -427,25 +476,28 @@ public void testIsolationForFallback() throws Exception {
   @Test(description = "Given a structure in which the classloader relationships form a graph, and two of the loaders "
       + "have a common parent, then delegation should properly load from the correct loaders")
   public void testGraphRelationshipWithCommonParent() throws Exception {
-    URL commonParentJarUrl = getTestJarUri("api").toURL();
+    URL commonParentJarUrl = getJarUri("cytodynamics-test-api").toURL();
     ClassLoader commonParent = new URLClassLoader(new URL[]{commonParentJarUrl}, null);
-    ClassLoader partialDelegation = new URLClassLoader(new URL[]{getTestJarUri("a").toURL()}, commonParent);
+    ClassLoader partialDelegation =
+        new URLClassLoader(new URL[]{getJarUri("cytodynamics-test-a").toURL()}, commonParent);
     ClassLoader loader = LoaderBuilder
         .anIsolatingLoader()
         .withOriginRestriction(OriginRestriction.allowByDefault())
-        .withClasspath(Collections.singletonList(getTestJarUri("b")))
+        .withClasspath(Collections.singletonList(getJarUri("cytodynamics-test-b")))
         .withParentRelationship(DelegateRelationshipBuilder.builder()
             .withDelegateClassLoader(commonParent)
             .withIsolationLevel(IsolationLevel.FULL)
             .addDelegatePreferredClassPattern("java.*")
-            // TODO fix: when loader's classloader doesn't match the parent classloader, Api annotation doesn't apply
+            // TODO fix: when loader's classloader doesn't match parent classloader, Api annotation doesn't apply
             .addDelegatePreferredClassPattern(TestInterface.class.getName())
             .build())
         .addFallbackDelegate(DelegateRelationshipBuilder.builder()
             .withDelegateClassLoader(partialDelegation)
-            // using NONE so that the class could potentially be loaded from here
-            .withIsolationLevel(IsolationLevel.NONE)
+            .withIsolationLevel(IsolationLevel.FULL)
```

Comment:
What's the reason for the switch from `NONE` to `FULL`?

Reply:
Before, NONE would allow all classes in the fallback to be loaded, including the `TestInterface` interface. However, this fallback should never be directly used to load `TestInterface`, since we should always go through the API classloader directly for that one. I'll clarify the comment below for that.